kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/ExpressionUtils.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (2023) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.internal.expressions;
+
+import java.math.BigDecimal;
+import java.util.Comparator;
+
+import io.delta.kernel.data.ColumnVector;
+import io.delta.kernel.types.*;
+
+import static io.delta.kernel.defaults.internal.DefaultKernelUtils.checkArgument;
+
+class ExpressionUtils {
+    private ExpressionUtils() {}
+
+    /**
+     * Utility method that calculates the nullability result from given two vectors. Result is
+     * null if at least one side is a null.
+     */
+    static boolean[] nullabilityEval(ColumnVector left, ColumnVector right) {
```

Comment:
nit: nullabilityEval is not very meaningful. something like `isAnyNull`?

Reply:
"nullability eval" does not convey the semantics at all. is it "left is null AND right is null" or "... OR.... " or ".... XOR ..." ?
"isAnyNull" conveys the semantics better.